api/bucketsd/client/client_test.go

```diff
@@ -1052,13 +1232,13 @@ func TestClose(t *testing.T) {
 }
 
 func setup(t *testing.T) (context.Context, *c.Client) {
-	//bconf := apitest.DefaultBillingConfig(t)
-	//apitest.MakeBillingWithConfig(t, bconf)
+	bconf := apitest.DefaultBillingConfig(t)
+	apitest.MakeBillingWithConfig(t, bconf)
 
 	conf := apitest.DefaultTextileConfig(t)
-	//billingApi, err := tutil.TCPAddrFromMultiAddr(bconf.ListenAddr)
-	//require.NoError(t, err)
-	//conf.AddrBillingAPI = billingApi
+	billingApi, err := tutil.TCPAddrFromMultiAddr(bconf.ListenAddr)
+	require.NoError(t, err)
+	conf.AddrBillingAPI = billingApi
```

Comment:
Re-enabled billing in the bucket tests. I had disabled this on some other PR and forgot to re-enabled.